### ISSUE TITLE
AlphabeticalImports is not working as expected.

### ISSUE THREAD

**Issue by josecanhelp** [tightenco/tlint#107]
This is how my imports are set up in `quicksand` right now [DeleteOldSoftDeletes.php](https://github.com/Jhnbrn90/quicksand/blob/feature/deleting-many-to-many-relationships/src/DeleteOldSoftDeletes.php):

```
use DateInterval;
use DateTime;
use Exception;
use Illuminate\Config\Repository as Config;
use Illuminate\Console\Command;
use Illuminate\Support\Collection;
use Illuminate\Support\Facades\Log;
use Illuminate\Support\Facades\DB;
use Illuminate\Support\Facades\Schema;
```

Tlint should squawk about Log and DB being out of alphabetical order, but running tlint against this file does not catch it.

**Comment by loganhenson** [comment]
This is working as intended (And should probably be explained along with many others, https://github.com/tightenco/tlint/issues/106)

It only squaks at the first segment alphabetical ordering, as that is what is required to make the code:
A) Aesthetically pleasing
B) Quick to find something in a top level namespace
C) It was super annoying/pedantic to have to sit and micro-swap stuff like `Log` and `DB` (especially when the lint only triggers on a single line, for simplicities sake, and that line is at the top of the imports block)

I'm open to discussion about changing it, but I believe this was the initial justification for the behavior.

**Comment by mattstauffer** [comment]
In my opinion it should be full order if possible.

**Comment by mattstauffer** [comment]
(My justification for this is: It should be predictable to find anything relative to anything, and full alpha is the most predictable.)

**Comment by loganhenson** [comment]
ok @mattstauffer (PRs welcome on this Jose, or I will get to it in my list 😄)
